testutil-server/src/main/java/io/spine/testing/server/blackbox/SingleTenantContext.java

```diff
@@ -33,7 +33,7 @@
 /**
  * Test fixture for single-tenant Bounded Contexts.
  */
-final class SingleTenantContext extends BlackBoxContext {
+final class SingleTenantContext extends BlackBox {
```

Comment:
Please consider renaming this class to `SingleTenantBlackBox` or `StBlackBox`. It could be also `BlackBoxSt` as well. This way the classes would go together in alpha sorting.